cmd/tfchaind/main.go

```diff
@@ -47,3 +54,53 @@ func SetupNetworks(name string) (daemon.NetworkConfig, error) {
 		return daemon.NetworkConfig{}, fmt.Errorf("Netork name %q not recognized", name)
 	}
 }
+
+// ConfigureUnlockConditions configures the unlock conditions,
+// as to define the unlock conditions and fulfillments.
+// For the most parts the configuration follows the standard Rivine unlock condition configuration,
+// but there are a couple of differences:
+//
+// * if the networkName equals "standard",
+//   the MultiSignatureCondition is only considered standard since block height 42000,
+//   giving peers on the standard network more or less 7 days, since 1.0.6 was released
+func ConfigureUnlockConditions(networkName string) {
+	if networkName != standardnet {
+		return // only apply the blockheight-based restrictions for standard net
+	}
+	// Overwrite the Rivine-standard MultiSignatureCondition with our wrapped version,
+	// as to ensure that this condition is only introduced in the
+	// standard tfchain since blockheight 42000
+	overwriteMultiSignatureConditionType()
+	// MultiSignatureFulfillment doesn't need to be overriden,
+	// as it is impossible to use it as long as there isn't an unspend output
+	// which uses the MultiSignatureCondition
+}
+
+func overwriteMultiSignatureConditionType() {
+	types.RegisterUnlockConditionType(types.ConditionTypeMultiSignature,
+		func() types.MarshalableUnlockCondition { return new(MultiSignatureCondition) })
```

Comment:
Did you mean: Marshallable ?

Reply:
I know it does not matter here, just caught my attention. btw, I just pasted here what google asked me when searching for the single 'l' version._

Reply:
Hmm, I don't use google normally, let me try that. Nope I don't get that, it seems to link me to some repos that do use the single 'l' version.